src/acl.c

```diff
@@ -2761,13 +2761,49 @@ void aclCommand(client *c) {
             addReplyBulkCString(c,"client-info");
             addReplyBulkCBuffer(c,le->cinfo,sdslen(le->cinfo));
         }
+    } else if (!strcasecmp(sub,"dryrun") && c->argc >= 4) {
+        struct redisCommand *cmd;
+        user *u = ACLGetUserByName(c->argv[2]->ptr,sdslen(c->argv[2]->ptr));
```

Comment:
are we sure we don't wanna just use the current user?

Reply:
My thought was that it's slightly safer to run this since you can test on a command that is disabled and has no passwords. It makes the command a little more verbose though.

Reply:
ok.. maybe it's indeed better, considering the user which we test this on may not have access to the ACL command.
and it's likely that the admin who defines the ACLs is the one which wants to test it.